src/hot-keys/RaceSpecificHotkeys/hotkeyData/nightelf/buildings.ts

```diff
@@ -35,7 +38,8 @@ const buildings =
             ...defaultFighterAbilities,
             Ability.Create('Eat Tree', 'btneattree', 'aeat', 'E', []),
             ...Ability.Defaults(2),
-            uproot
+            uproot,
```

Comment:
if root is an `UnHotkey` , we do not need this here

Reply:
yep, fixed!